transforms/runTransform.js

```diff
@@ -10,13 +10,15 @@ import fs from 'fs/promises'
 import { walkComponents } from './walkComponents.js'
 import { shadowToLight } from './shadowToLight/index.js'
 import { syntheticToNative } from './syntheticToNative/index.js'
+import { htmlTemplateCleanup } from './htmlTemplateCleanup/index.js'
 
 const include = ['.js', '.ts']
 const exclude = ['__tests__', '.stories.ts']
 
 const transforms = {
   'shadow-to-light': shadowToLight,
-  'synthetic-to-native': syntheticToNative
+  'synthetic-to-native': syntheticToNative,
+  'html-template-cleanup': htmlTemplateCleanup
```

Comment:
I'm open to renaming this, I couldn't think of a good name for a general codemod to fix HTML errors/warnings.